### DISCUSSION TITLE
How to add a new section in `lualine`?

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [LazyVim/LazyVim#537] [Q&A] [answered]
I followed the [instructions](https://www.lazyvim.org/configuration/examples) to add a new section in `lualine`:

```lua
  {
    "nvim-lualine/lualine.nvim",
    event = "VeryLazy",
    opts = function(_, opts)
      table.insert(opts.sections.lualine_x, "😄")
    end,
  },
```
But the smiling face does not show at the status line. What do I miss here?

**Comment by ChenZhongPu** [comment, marked as answer]
I found that it must be wrapped inside a function:

```lua
    opts = function(_, opts)
      table.insert(opts.sections.lualine_x, function()
        return "😄"
      end)
```

**Comment by nipuna-perera** [comment]
If the example is wrong, shouldn't this discussion be kept open till it's
fixed?
